CMakeLists.txt

```diff
@@ -324,6 +325,7 @@ set(BOOST_INCLUDE_LIBRARIES
     context
     date_time
     filesystem
+    math
```

Comment:
math is a header only lib and comes with the `headers` target from system boost. Bundled boost does not create the headers target automatically that's why we do it manually:
`set(BOOST_HEADER_ONLY crc circular_buffer multi_index random uuid variant)` <- this is where math needs to be added so it gets added to our custom headers target.